src/components/Form/formStyling.tsx

```diff
@@ -30,7 +30,8 @@ export const StyledCard = styled(Paper)<{ $compact?: boolean }>`
     width: 100%;
     display: flex;
     padding: 2em;
-    height: 80%;
+    height: auto;
+    margin-bottom: 2px;
```

Comment:
Taking a look at those screenshots, I think we can space them out even a little more. Can we change this to `margin-top: 5px`? Also at the top so we get some space between the top card and the component above it.